src/main/java/com/iexec/sms/config/OpenApiConfig.java

```diff
@@ -16,29 +16,26 @@
 
 package com.iexec.sms.config;
 
+import com.iexec.sms.utils.version.VersionService;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import springfox.documentation.builders.PathSelectors;
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @Configuration
-@EnableSwagger2
-public class SwaggerConfig {
+public class OpenApiConfig {
 
-    /*
-     *
-     * Swagger link:
-     * http://localhost:13300/swagger-ui.html
-     * */
-    @Bean
-    public Docket api() {
-        return new Docket(DocumentationType.SWAGGER_2)
-                .select()
-                .apis(RequestHandlerSelectors.any())
-                .paths(PathSelectors.any())
-                .build();
+    private final VersionService versionService;
+
+    public OpenApiConfig(VersionService versionService) {
```

Comment:
Can you please add the url to access the docs page in a comment.

Reply:
Good idea! See https://github.com/iExecBlockchainComputing/iexec-sms/pull/82/commits/e7b6ef3284e91f0f3bf54eb9f0196d9d1375f1ea.